components/studio/bin/hab-studio.sh

```diff
@@ -1091,43 +1120,23 @@ unmount_filesystems() {
   # currently mounted. You know, so you can run this all day long, like, for
   # fun and stuff.
 
-  if $bb mount | $bb grep -q "on $HAB_STUDIO_ROOT/src type"; then
-    $bb umount $v -l $HAB_STUDIO_ROOT/src
-  fi
+  try_umount $v -l $HAB_STUDIO_ROOT/src
 
   local studio_artifact_path
   studio_artifact_path="${HAB_STUDIO_ROOT}${HAB_CACHE_ARTIFACT_PATH}"
-  if $bb mount | $bb grep -q "on $studio_artifact_path type"; then
-    $bb umount $v -l $studio_artifact_path
-  fi
+  try_umount $v -l $studio_artifact_path
 
-  if $bb mount | $bb grep -q "on $HAB_STUDIO_ROOT/run type"; then
-    $bb umount $v $HAB_STUDIO_ROOT/run
-  fi
+  try_umount $v $HAB_STUDIO_ROOT/run
 
-  if $bb mount | $bb grep -q "on $HAB_STUDIO_ROOT/sys type"; then
-    if [ -z "${KRANGSCHNAK+x}" ]; then
```

Comment:
For the moment we'll need to preserve this if/else case with the slightly different `umount` args. It's a temporary fix to handle `/sys` differently if it was set up in a user namespace or not.

Reply:
Good catch; I didn't mean to change that.